request.schema.json

```diff
@@ -12,6 +12,105 @@
             "type": "string",
             "pattern": "^data:image/[^;]+;base64,[a-zA-Z0-9+/=]+$"
         },
+        "highChartsData": {
+            "description":"Data extracted from the highchart",
```

Comment:
```suggestion
            "description":"Data extracted from the chart",
```

Reply:
I'd advise against this if it is in fact just the highchart format.